### DISCUSSION TITLE
Only display the first letter of each day (e.g. L M M J V S D)

### DISCUSSION THREAD

**Discussion by flavinsh** [liamcain/obsidian-calendar-plugin#315] [Q&A]
Hi,

How can I display only the **first letter** of each day in the calendar pane ?
Using the French locale, the day row looks very cluttered (see below).

**Current behaviour:** 
> LUN. MAR. MER. JEU. VEN. SAM. DIM

**Desired behaviour:**
> L M M J V S D

I guess that this could be achieved by a simple CSS snippet.
Unfortunately, after searching extendedly, I couldn't find a solution to my problem (sorry: noob here).

Many thanks in advance

Edit: I am using _Minimal Theme_, if that matters.

**Comment by flavinsh** [comment]
If it helps, I have found the following items using Developper Tools :

<img width="811" alt="image" src="https://github.com/liamcain/obsidian-calendar-plugin/assets/82162075/36ff4571-6735-4dec-a03c-2df6a094eec0">

If I replace ```lun.```by ```L```, ```mar.```by ```M```(etc), it works and each day displays as one letter, as desired.

However, it resets back to default after restarting Obsidian.

**Hower can I make this modification permanent ?** 

I guess that it should be possible using a CSS snippet, for instance.

**Comment by MolaveM** [comment]
I would love to learn how to do this permanently, too. The sidebar in which I choose to place Calendar is very narrow, so the weekday headings run together, as in, SUNMONTUEWEDTHUFRISAT. 

If I could reduce the characters to one or two (like the primary screenshot on this GitHub page), readability would increase dramatically, as in, S  M  T  W  H  F  S. 

Thank you very much.

**Comment by giftopherz** [comment]
Hi! I just tried it with a little tweak to the main.js file in VSCode. 
I'm not quite proficient but this did it

Open main.js (from the plugins calendar folder) in vscode (or whatever code editor you have, note pad can do it but it'll take longer to find)
Line 2094, that's the content of function getDaysOfWeek

```
function getDaysOfWeek(..._args) {
    // Grab the short name and take only the first character
    return window.moment.weekdaysShort(true)**.map(d => d.charAt(0))**;
}
```

you just have to add .map(d => d.charAt(0)) after "(true)"

**Comment by MolaveM** [comment]
Wow it works! I just have to remember to make that change again whenever the Calendar plugin updates. No worries on that. I also had to remove those asterisks, hehe, which were inserted because markdown I guess? Thanks for the simple solution.
